aiida/tools/importexport/dbexport/__init__.py

```diff
@@ -658,16 +659,22 @@ def export_tree(what,
                 else:
                     groups_uuid[str(res[0])] = [str(res[1])]
 
+    #######################################
+    # Final check for unsealed ProcessNodes
+    #######################################
+    process_nodes = set()
+    for node_pk, content in export_data.get(NODE_ENTITY_NAME, {}).items():
+        if content['node_type'].startswith('process.'):
+            process_nodes.add(node_pk)
```

Comment:
Do we not have the node instances that are going to be exported at some point? If so, it would be better to keep that as a set and pass them to `check_process_nodes_sealed` such that there they won't have to be reloaded again. Never mind: I just saw the implementation of the function that you filter on the pks. I guess this is fine then